test/test_QtMultimedia.py

```diff
@@ -1,4 +1,11 @@
-from PySide2 import QtMultimedia
+from python_qt_binding import QT_BINDING
```

Comment:
We don't have a module called `python_qt_binding`. I think this script would fail in your system. To get `QT_BINDING`, you need something like this.
```
QT_BINDING=None
try:
  import PySide2
  QT_BINDING='pyside'
except:
  try:
    import PyQt5
    QT_BINDING='pyqt'
  except:
    raise ImportError('Qt binding name "%s" is unknown.' % QT_BINDING) 
```

Reply:
You'd need to install the python_qt_binding module:

`pip3 install python_qt_binding`

Although I guess you prefer not relying on it

Reply:
Yes, I don't think we need it. If there is no problem, I will remove it.

Reply:
If we do as you say, which is fine to me, we should probably do it as follows so it is clearer:

```
try:
  from PySide2 import QtMultimedia
  QT_BINDING='pyside'
except:
  try:
    from PyQt5 import QtMultimedia
    QT_BINDING='pyqt'
  except:
    QT_BINDING='pyside, pyqt'
    raise ImportError('Qt bindings "(%s)" not found.' % QT_BINDING) 
```

I can make the change if you want

Reply:
I tested my code above following try/except and works equally fine

Reply:
> I can make the change if you want

Thank you. It is done.

Reply:
Sorry, I thought you wanted to take `python_qt_binding` out, and created a new pull request #10 about that.

Feel free to close or merge as you think it is better